packages/create-keystone-app/example-projects/nuxt/index.js

```diff
@@ -0,0 +1,29 @@
+const { Keystone } = require('@keystone-alpha/keystone');
+const { Text } = require('@keystone-alpha/fields');
+const { GraphQLApp } = require('@keystone-alpha/app-graphql');
+const { AdminUIApp } = require('@keystone-alpha/app-admin-ui');
+const { NuxtApp } = require('@keystone-alpha/app-nuxt');
+
+const nuxtConfig = {
+  srcDir: 'src',
+  buildDir: 'dist',
+};
+
+const { MongooseAdapter: Adapter } = require('@keystone-alpha/adapter-mongoose');
```

Comment:
This needs some magic comments to ensure everything works correctly when generated via the CLI. See the `static` project for an example: https://github.com/keystonejs/keystone-5/blob/5fc2bd597ed8f6b0dfaf7761e1f20bc2ad14caf5/packages/create-keystone-app/example-projects/todo/index.js#L7-L10